extensions-contrib/orc-extensions/src/main/java/io/druid/data/input/orc/OrcHadoopInputRowParser.java

```diff
@@ -132,6 +138,23 @@ private List getListObject(ListObjectInspector listObjectInspector, Object listO
     return list;
   }
 
+  private void getMapObject(String parentName, MapObjectInspector mapObjectInspector, Object mapObject, Map<String, Object> parsedMap)
+  {
+    if (mapObjectInspector.getMapSize(mapObject) < 0) {
+      return;
+    }
+    String prefix = parentName + "_";
+    Map objectMap = mapObjectInspector.getMap(mapObject);
+    PrimitiveObjectInspector key = (PrimitiveObjectInspector) mapObjectInspector.getMapKeyObjectInspector();
+    PrimitiveObjectInspector value = (PrimitiveObjectInspector) mapObjectInspector.getMapValueObjectInspector();
+    for (Iterator itr = objectMap.entrySet().iterator(); itr.hasNext(); ) {
```

Comment:
There's a TeamCity inspection error here:

```
 extensions-contrib/orc-extensions/src/main/java/io/druid/data/input/orc
 OrcHadoopInputRowParser.java (1)
150: getMapObject() for loop replaceable with 'foreach'
```

Reply:
@jon-wei it's already fixed.